api/src/main/java/org/openmrs/LocationAttributeType.java

```diff
@@ -30,6 +32,9 @@
 @Entity
 @Table(name = "location_attribute_type")
 @Audited
+@AttributeOverrides(value = {
+	@AttributeOverride(name = "description", column = @Column(name = "description", length = 1024))
```

Comment:
Did you intentionally give some descriptions a size of 1024 but others a size of 255?